themes/base.theme.bash

```diff
@@ -270,8 +272,10 @@ function git_prompt_vars {
   [[ "${status}" =~ ${ahead_re} ]] && SCM_BRANCH+=" ${SCM_GIT_AHEAD_CHAR}${BASH_REMATCH[1]}"
   [[ "${status}" =~ ${behind_re} ]] && SCM_BRANCH+=" ${SCM_GIT_BEHIND_CHAR}${BASH_REMATCH[1]}"
 
+  SCM_PREFIX=${SCM_GIT_STASH_CHAR_PREFIX:-$SCM_GIT_STASH_CHAR_PREFIX}
```

Comment:
What do these two lines do?

Reply:
They do nothing. I was playing around with default values and then I forgot to clean it.